app/requests.py

```diff
@@ -0,0 +1,24 @@
+class RegisterRequest:
+    def __init__(self, name: str = None, email: str = None, password: str = None, ):
+        self.name = name
+        self.email = email
+        self.password = password
+
+    def to_dict(self):
```

Comment:
Moze jakas metoda utils importowana? Bo drugie miejsce gdzie to wystepuje

Reply:
przydałoby się, ale moim zdaniem to na oddzielnego taska jak już

Reply:
dobra, nvm. znalazłem rozwiązanie, zmieniłem